csp/decorators.py

```diff
@@ -10,8 +18,26 @@ def _wrapped(*a, **kw):
     return _wrapped
 
 
-def csp_update(**kwargs):
-    update = dict((k.lower().replace('_', '-'), v) for k, v in kwargs.items())
+def csp_reorder(order):
```

Comment:
Big thing left to do is clarify how the decorators should work and write tests for them.